<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Steve Xu</author>
 <class>FinanceMgrClass</class>
 <widget class="QMainWindow" name="FinanceMgrClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FinanceMgr</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionInsert"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDel"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionStrike"/>
    <addaction name="actionFinal"/>
    <addaction name="actionTransfer"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>报表</string>
    </property>
    <addaction name="actionCash"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="actionCheck"/>
    <addaction name="actionBudget"/>
    <addaction name="actionIncome"/>
    <addaction name="actionBankAccount"/>
    <addaction name="actionDepartment"/>
    <addaction name="actionTitle"/>
    <addaction name="actionDetailTitle"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_6"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>操作</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInsert"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDel"/>
   <addaction name="separator"/>
   <addaction name="actionImport"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionStrike"/>
   <addaction name="actionFinal"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>拷贝</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="icon">
    <iconset resource="financemgr.qrc">
     <normaloff>:/Icon/Resources/add2.png</normaloff>:/Icon/Resources/add2.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="financemgr.qrc">
     <normaloff>:/Icon/Resources/edit.png</normaloff>:/Icon/Resources/edit.png</iconset>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="financemgr.qrc">
     <normaloff>:/Icon/Resources/delete.png</normaloff>:/Icon/Resources/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionDepartment">
   <property name="text">
    <string>部门管理</string>
   </property>
  </action>
  <action name="actionBankAccount">
   <property name="text">
    <string>帐号管理</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>请款单管理</string>
   </property>
  </action>
  <action name="actionBudget">
   <property name="text">
    <string>预算管理</string>
   </property>
  </action>
  <action name="actionTitle">
   <property name="text">
    <string>科目管理</string>
   </property>
  </action>
  <action name="actionDetailTitle">
   <property name="text">
    <string>详细科目管理</string>
   </property>
  </action>
  <action name="actionIncome">
   <property name="text">
    <string>收入管理</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="financemgr.qrc">
     <normaloff>:/Icon/Resources/box_download_48.png</normaloff>:/Icon/Resources/box_download_48.png</iconset>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="financemgr.qrc">
     <normaloff>:/Icon/Resources/box_upload_48.png</normaloff>:/Icon/Resources/box_upload_48.png</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actionCash">
   <property name="text">
    <string>现金收支表</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>银行存款流量表</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>特别预算表</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>预算差异表</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>请款明细表</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>冲账明细表</string>
   </property>
  </action>
  <action name="actionStrike">
   <property name="icon">
    <iconset resource="financemgr.qrc">
     <normaloff>:/Icon/Resources/app_48.png</normaloff>:/Icon/Resources/app_48.png</iconset>
   </property>
   <property name="text">
    <string>冲账</string>
   </property>
  </action>
  <action name="actionFinal">
   <property name="icon">
    <iconset resource="financemgr.qrc">
     <normaloff>:/Icon/Resources/accepted_48.png</normaloff>:/Icon/Resources/accepted_48.png</iconset>
   </property>
   <property name="text">
    <string>结案</string>
   </property>
  </action>
  <action name="actionTransfer">
   <property name="icon">
    <iconset resource="financemgr.qrc">
     <normaloff>:/Icon/Resources/arrow_right_green_48.png</normaloff>:/Icon/Resources/arrow_right_green_48.png</iconset>
   </property>
   <property name="text">
    <string>预算资料结转</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="financemgr.qrc">
     <normaloff>:/Icon/Resources/Print.png</normaloff>:/Icon/Resources/Print.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
   <property name="toolTip">
    <string>打印</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="financemgr.qrc"/>
 </resources>
 <connections/>
</ui>
